src/services/StructuralMetadataUtils.js

```diff
@@ -365,7 +373,11 @@ export default class StructuralMetadataUtils {
       findSpanItem(wrapperSpans.after, allItems);
     }
 
-    return validHeadings;
+    // Filter out the duplicated Section labels
+    let uniqueValidHeadings = validHeadings.filter(
+      (heading, i) => validHeadings.indexOf(heading) === i
+    );
+    return uniqueValidHeadings;
   }
```

Comment:
So I'm wondering why there are duplicated Section labels?  They shouldn't be allowed to exist...or at least that's the unique identifier for each item in the data structure.  If this is just a double check, can you note in the comment something to the effect that duplicate labels aren't allowed, but just in case one snuck in, this will remove them.

Basically if duplicate labels start sneaking in, this whole front end logic might need to be re-factored.

Reply:
This happened because I added different timepans with the same label, before I got to know the labels are unique. I think this should be removed, because it is unnecessary load on the code. Should I remove it?

Reply:
Maybe I'm commenting too late, but I can easily imagine use cases where labels would be duplicated within a singe structure. E.g. ```
Act 1
    Intro
Act 2
    Intro
```

Reply:
It's good to address this now, because I'm pretty sure that will not work.  There needs to be some unique identifier for each item in the data structure element.  I think I posed this question a while ago, but maybe not loud enough...which totally could be the case.  If this is the case, we may need to re-think a lot of this existing code.

Reply:
One way to do this might be there'd be an `id` with each entry.  And when you add a new one, it makes a post to the server, creates the new heading or timespan, and returns the submitted values with an id as part of the JSON structure?    @bkeese Is this a pain with how the back end stores the XML, or JSON values?

Or maybe we would have to target by which parent a timespan lives in, and then a timespan's label would have to be unique?